lib/ASRouterTargeting.jsm

```diff
@@ -140,6 +140,9 @@ const TargetingGetters = {
       totalDevices: Services.prefs.getIntPref("services.sync.numClients", 0),
     };
   },
+  get xpinstallEnabled() {
+    return Services.prefs.getBoolPref("xpinstall.enabled", true);
```

Comment:
The only reason I'd prefer to leave it as fallback to `true` is so that the definition in targeting-attributes.md is consistent in all cases. What do you think?